tests/testthat/test-gaussian.R

```diff
@@ -54,9 +66,29 @@ test_that("we can approximate the closed form ridge regression solution", {
                        intercept = FALSE,
                        thresh = 0.00001,
                        maxit = 1000)
+  
+  batch_fit <- sgdnet(x, y,
+                       alpha = 0,
+                       lambda = sd_y*lambda/n,
+                       intercept = FALSE,
+                       thresh = 0.00001,
+                       maxit = 1000,
+                       batchsize = 10)
+
+  batch_full <- sgdnet(x, y,
+                      alpha = 0,
+                      lambda = sd_y*lambda/n,
+                      intercept = FALSE,
+                      thresh = 0.000001,
+                      maxit = 1000,
+                      batchsize = 500)
```

Comment:
Can we set this 500 to `n` or to `NROW(x)` to make the intent (full gradient) a bit clearer?